nautobot/core/ui/object_detail.py

```diff
@@ -452,7 +452,7 @@ def _get_table_add_url(self, context):
         This method determines the URL for adding a new object to the table. It checks if the user has
         the necessary permissions and creates the appropriate URL based on the specified add button route.
         """
-        obj = context["obj"]
+        obj = context.get("obj") or context.get("object")
```

Comment:
Thought we always had `obj` as a context variable :(

Reply:
Yeah still a number of differences between the context variables in legacy view patterns and those in NautobotUIViewSet.

Reply:
Let's create a `get_obj_from_context`  helper function that we can use throughout then.